bin/run.sh

```diff
@@ -17,44 +17,66 @@
 
 # If any required arguments is missing, print the usage and exit
 if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
-    echo "usage: ./bin/run.sh exercise-slug path/to/solution/folder/ path/to/output/directory/"
-    exit 1
+  echo "usage: ./bin/run.sh exercise-slug path/to/solution/folder/ path/to/output/directory/"
+  exit 1
 fi
 
+set -eu
+
+root_dir=$(dirname "$(dirname "$(realpath "$0")")")
 slug="$1"
 solution_dir=$(realpath "${2%/}")
 output_dir=$(realpath "${3%/}")
 results_file="${output_dir}/results.json"
 
+echo "Copying dependencies..."
+cd packages
+mkdir -p "$solution_dir"/build
+cp -r "$root_dir"/packages/build/packages "$solution_dir"/build/packages
+
+sanitise_gleam_output() {
+  grep -vE \
+    -e "^Downloading packages" \
+    -e "^ Downloaded [0-9]+ packages in [0-9]\.[0-9]+s" \
+    -e "^ Downloaded [0-9]+ packages in [0-9]\.[0-9]+s" \
```

Comment:
Will these lines be printed anyway even if dependencies were copied from the packages project?

Reply:
I don't think so, no. This is likely reundant now, I'll remove it

Reply:
I was wrong, it may still get printed